docs/setup/feeds.md

```diff
@@ -0,0 +1,17 @@
+# Feeds
```

Comment:
I added some skeleton docs because I couldn't resist. We have quite a few places that need updating when a new service is added so I thought this would save some time.

Reply:
Fleshed them out now:)